visualcube.php

```diff
@@ -200,7 +200,7 @@
 					case 'x' : $rtn_[$i][0] = 0; break;
 					case 'y' : $rtn_[$i][0] = 1; break;
 					case 'z' : $rtn_[$i][0] = 2; break;
-					default : continue;
+					default : break;
```

Comment:
The original `continue` printed a warning on the rendered page. This fixes it. Maybe dependent on the PHP version?